nuxt.config.ts

```diff
@@ -126,7 +126,8 @@ const config: Configuration = {
       }
     ],
     'nuxt-svg-loader',
-    'nuxt-purgecss'
+    'nuxt-purgecss',
+    ['vue-scrollto/nuxt', { duration: 1000, offset:-72 }]
```

Comment:
どうやらリントがひっかかってるぽいので、修正お願いします！
```suggestion
    ['vue-scrollto/nuxt', { duration: 1000, offset: -72 }]
```

Reply:
解消しました！お手数おかけしました